docs/build-tools/overview.md

```diff
@@ -31,14 +32,20 @@ compiler plugin and `-Yrangepos` option enabled.
 
 ## Goto library dependencies
 
-**✅**: it is possible to navigate Scala+Java library dependencies using "Goto
```

Comment:
What do you think about replacing the checkboxes with words?

✅ becomes "Semi-automatic"

✔️ becomes "Automatic"

Reply:
We can do that, I was just thinking the idea was to use those ticks?

Reply:
Seeing the table now, I'm not convinced which one of ✔️ and ✅ intuitively means more complete than the other. I think the words "Automatic" and "Semi-automatic" are more self-explanatory.

Reply:
Changed - let me know what you think